templates/usbhid.conf.j2

```diff
@@ -0,0 +1,2 @@
+{{ ansible_managed }}
+blacklist usbhid
```

Comment:
Better put the logic into the template rather than the tasks.

```suggestion
{% if not xibo_enable_hd_module %}
blacklist usbhid
{% endif %}
```